include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.h

```diff
@@ -101,22 +101,22 @@ class TimeIntBDF : public TimeIntBDFBase<Number>
   print_iterations() const = 0;
 
   bool
-  print_solver_info() const;
+  print_solver_info() const override final;
 
 protected:
-  virtual void
+  void
```

Comment:
I think it is not correct to remove `virtual` here?

Reply:
It is correct, `override` implies `virtual`.